src/App.js

```diff
@@ -1,40 +1,59 @@
 import './App.css';
 import freeCodeCampLogo from './img/freecode.png';
 import Boton from './componentes/Boton';
+import Pantalla from './componentes/Pantalla';
+import BtnClear from './componentes/BtnClear';
+import { useState } from 'react';
+import {evaluate} from 'mathjs'
```

Comment:
You can set a [prettier format](https://dev.to/knowankit/setup-eslint-and-prettier-in-react-app-357b) aiming to have the same style in all the project